pkg/route/graph/edges.go

```diff
@@ -18,12 +18,14 @@ const (
 
 // AddRouteEdges adds an edge that connect a service to a route in the given graph
 func AddRouteEdges(g osgraph.MutableUniqueGraph, node *routegraph.RouteNode) {
-	syntheticService := &kapi.Service{}
-	syntheticService.Namespace = node.Namespace
-	syntheticService.Name = node.Spec.To.Name
+	for _, svc := range node.Spec.To {
```

Comment:
This is going to screw up prioritization and grouping, but I don't have a good answer right now.